src/components/tabs/tabs.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
+import { Tab } from "@headlessui/react";
+
+export type TabItem = {
+  label: string;
+  content: React.ReactNode; // or JSX.Element
+};
+
+export type TabsProps = {
+  tabs: TabItem[];
+};
+
+function Tabs({ tabs }: TabsProps) {
+  const [selectedIndex, setSelectedIndex] = useState(0);
+
+  return (
+    <Tab.Group
+      selectedIndex={selectedIndex}
+      onChange={setSelectedIndex}
+      defaultIndex={0}
+    >
+      <Tab.List className="mb-5 max-w-[354px] w-full h-10 flex">
+        {tabs.map((tab, index) => (
+          <Tab
+            key={index}
+            className={({ selected }) =>
+              `text-base outline-none font-rubik text-[#121212] font-normal leading-5 p-2.5 border-b-2 w-1/2 ${
```

Comment:
text-[#121212] needs to be replaced by the class defined in the config file (text-black)